discord-bot/src/interactions/application_commands/matchmaking.rs

```diff
@@ -230,6 +259,143 @@ impl InteractionHandler for MatchmakingCommandHandler {
+                    }
+                }
+
+                // TODO: Check if opponent is the bot, because that would be invalid.
+                if !reporter_is_member {
+                    return Err(anyhow!("user isn't part of the lobby"));
+                } else if !opponent_is_member {
+                    return Err(anyhow!("opponent isn't part of the lobby"));
+                }
+
+                let options: Vec<CommandDataOption>;
+                match subcommand.value.clone() {
+                    CommandOptionValue::SubCommand(x) => options = x,
+                    _ => options = Vec::<CommandDataOption>::new(),
+                }
+
+                let wins_option_value = options
+                    .get(1)
+                    .ok_or_else(|| anyhow!("could not get wins option"))?
+                    .value
+                    .clone();
+
+                let loses_option_value = options
+                    .get(2)
+                    .ok_or_else(|| anyhow!("could not get loses option"))?
+                    .value
+                    .clone();
+
+                let wins: i64 = match wins_option_value {
+                    CommandOptionValue::Integer(x) => x as i64,
+                    _ => 0 as i64,
+                };
+
+                let loses: i64 = match loses_option_value {
+                    CommandOptionValue::Integer(x) => x as i64,
+                    _ => 0 as i64,
+                };
+
+                // TODO: Refactor all this message stuff to reusable functions with parameters?
+                // TODO: Add an embed with options for the opponent to accept or dispute the score report.
+                // TODO: Format the message so it tags both the User & Opponent.
+                //       And show the wins for both of them in a nice looking way.
+                let guild_settings = self
+                    .utils
+                    .get_guild_settings(
+                        data.command
+                            .guild_id
+                            .ok_or_else(|| anyhow!("command cannot be used in a DM"))?,
+                    )
+                    .await?;
+                let channel;
+                if let Some(cid) = guild_settings.channel_id {
+                    //
+                    // TODO: make sure that the channel actually exists.
+                    //
+                    channel = cid.into_id();
+                } else {
+                    channel = data
+                        .interaction
+                        .channel_id
+                        .ok_or_else(|| anyhow!("command was not run in a channel"))?;
+                }
+                let _followup = self
+                    .utils
+                    .http_client
+                    .interaction(self.utils.application_id)
+                    .create_followup(data.interaction.token.as_str())
+                    .content(format!("Sent a score report in <#{}>", channel).as_str())?
```

Comment:
This message is a bit misleading because no score report was sent in the default matchmaking channel. I would prefer this line be removed until that logic is implemented.
```suggestion
```